src/assets/components/details.jsx

```diff
@@ -0,0 +1,39 @@
+import { useParams } from 'react-router-dom'
+import { useState, useEffect } from 'react'
+import '../../index.css'
+import { BackButton } from './BackButton';
+
+export const Details = () => { 
+  const [movie, setMovie] = useState({});
+  const apiKey = 'aea11e511c691453635b1da985972186';
+  const { id } = useParams();
+
+  // Fetches the details of a specific movie using the movie's id and the API key
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=${apiKey}&language=en-US`)
+      .then(res => res.json())
+      .then(data => setMovie(data))
+      .catch(error => alert("Error fetching data:" + error));
+  }, [id]);
+
+  // Converting the movie's audience score to an integer, so the decimals could be rounded later with the toFixed method
+  let voteAverage = parseFloat(movie.vote_average)
```

Comment:
I didn't seem to convert this to an integer and did this instead
voteAverage.toFixed(1) (as I see you have below in line 30)